### ISSUE TITLE
datasette-leaflet-freedraw causes 'no such table: geometry_columns' when enrichment uses spatialite

### ISSUE THREAD

**Issue by hcarter333** [datasette/datasette-enrichments#39]
tldr; the geocoding enrichment still works although the error messages discussed below is created, so this isn't critical.

I forked and then modified the datasette-enrichments-opencage package to use the google maps API instaed of opencage. The new repository is called [enrichments-gmap-geocode](https://github.com/hcarter333/enrichments-gmap-geocode).

When the datasette-leaflet-freedraw plugin is installed, this same [issue](https://github.com/simonw/datasette-leaflet-freedraw/issues/11) from the datasette-leaflet-freedraw is caused.

Uninstalling the plugin resolves the issue, but... my larger datasette ecosystem will need both the enrichment and the plugin in the following use case:
### Use Case
One table stores ham radio QSO (contact) logs including the location of the transmitting and receiving ham radio stations (lat and lng.) This table will is used along with the [freedraw to search for QSOs](https://copaseticflow.blogspot.com/2023/10/things-i-learned-querying-qsos-in.html) at different transmit or receive locations (click through for video use case demo).

[![IMAGE ALT TEXT HERE](https://img.youtube.com/vi/snjI5oJNQUo/0.jpg)](https://www.youtube.com/watch?v=snjI5oJNQUo)

or statically:
![image](https://github.com/datasette/datasette-enrichments/assets/363004/b94d385f-5b4d-40da-beb9-cef04c4084a9)

Another table (perhaps just the first one mentioned above?) will contain callsigns for new QSOs. These callsigns will be passed through two enrichments in turn, one that uses qrz.com to lookup the callsign address, and then [enrichments-gmap-geocode](https://github.com/hcarter333/enrichments-gmap-geocode) to geocode the station locations.

Is there a resolution other than uninstalling the freedraw plugin?

To recreate:
Clone and install the gmaps API [enrichment](https://github.com/hcarter333/enrichments-gmap-geocode). To install I used 
`pip install -e '.[test]'`

at the top level of the cloned repo

Then, create an addresses table:
```
Club_Station,Address
ARRL,"connecticut,USA"
W6BB,"Berkeley,CA,USA"
W6YX,"Palo Alto,CA,USA"
```
that is loaded into a sqlite db in any manner you like. (I used `csvs-to-sqlite ../test_addr.csv gm_test.db`)

Then, start datasette with the following command line:
`python3 -m datasette gm_test.db --metadata qso_loc.yml --load-extension=/usr/lib/x86_64-linux-gnu/mod_spatialite.so --template-dir plugins/templates --root`

Go through the normal flow to enrich the address data and the error will appear.

Also!? The data will still be enriched, so may it doesn't matter?

Here's the error:
![image](https://github.com/datasette/datasette-enrichments/assets/363004/31d8f6fd-5496-4367-942b-43d057db8e4d)

But the geocoding actually still worked:
![image](https://github.com/datasette/datasette-enrichments/assets/363004/8bf9d308-7e8d-48a2-ac97-8442e4bdc140)
